addons/mail/data/mail_channel_demo.xml

```diff
@@ -16,6 +16,19 @@
             <field name="description">Research and development discussion group</field>
         </record>
 
+        <!-- TODO remove demo users -->
+        <record model="mail.channel" id="channel_rtc_1">
```

Comment:
Not sure those demo data adds something to Odoo demo mode...

Reply:
Only useful while we were testing, we will drop changes to demo before merging.